tests/dummy/app/templates/comps/form.hbs

```diff
@@ -150,13 +150,22 @@
   placeholder="Placeholder"
 />
 
+<Subheading @title="Form::HtmlInput" />
```

Comment:
Nitpick: move this up so it's below `FileInput`, so everything is in alphabetical order. Makes it easier to track down component examples.